.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.6.0
```

Comment:
Sorry, could you remove this file from the repo?

Reply:
It won't actually lock the apps that use the gem. It works just as a "hint" for developers. I've found it quite useful when there are issues between Rubies that some developers see and others don't.

It's not a big deal, and can remove it if you wish. Should I still remove it?

Reply:
I'd prefer it to be removed.